BULLETIN.md

```diff
@@ -3,20 +3,23 @@ Check out *https://agpt.co*, the official news & updates site for Auto-GPT!
 The documentation also has a place here, at *https://docs.agpt.co*
 
 # For contributors 👷🏼
-Since releasing v0.3.0, whave been working on re-architecting the Auto-GPT core to make it more extensible and make room for structural performance-oriented R&D.
-
 Check out the contribution guide on our wiki:
 https://github.com/Significant-Gravitas/Auto-GPT/wiki/Contributing
 
-# 🚀 v0.4.3 Release 🚀
-We're happy to announce the 0.4.3 maintenance release, which primarily focuses on refining the LLM command execution,
-extending support for OpenAI's latest models (including the powerful GPT-3 16k model), and laying the groundwork 
-for future compatibility with OpenAI's function calling feature.
-
-Key Highlights:
-- OpenAI API Key Prompt: Auto-GPT will now courteously prompt users for their OpenAI API key, if it's not already provided.
-- Summarization Enhancements: We've optimized Auto-GPT's use of the LLM context window even further.
-- JSON Memory Reading: Support for reading memories from JSON files has been improved, resulting in enhanced task execution.
-- Deprecated commands, removed for a leaner, more performant LLM: analyze_code, write_tests, improve_code, audio_text, web_playwright, web_requests
-## Take a look at the Release Notes on Github for the full changelog!
-https://github.com/Significant-Gravitas/Auto-GPT/releases
+# 🚀 v0.4.4 Release Highlights! 🚀
+### GPT-4 is back!
+
+Following OpenAI's GPT-4 GA announcement, SMART_LLM now defaults to GPT-4.
```

Comment:
```suggestion
Following OpenAI's GPT-4 General Availability announcement, Auto-GPT will use GPT-4
by default in its main run-loop.
```

Reply:
Updated, but kept the reference to SMART_LLM as the users may want to know how to change this.